controllers/admin/AdminTranslationsController.php

```diff
@@ -23,6 +23,9 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
+
+declare(strict_types=1);
```

Comment:
Do not add strict type on old files Especially in legacy ones which are clearly not adapted to this

We only add strict type on new files to limit the unexpected side effects